docs/README.md

```diff
@@ -69,7 +69,12 @@ papr.updateOne ~ 826.21 ops/sec
 mongoose.updateOne ~ 766.84 ops/sec
 ```
 
-This benchmark can be run locally with `yarn benchmark`, by default the benchmark tool will use `mongodb://localhost:27017`; however you can pass your own url by using the argument `--url` or `-u`.
+This benchmark can be run locally with `yarn benchmark`. See below for available arguments.
+
+| Parameter  | Accepted Type  | Default  | Description  |
+|---|---|---|---|
+| `--db` `-d`  | *string*  | `benchmark${UNIXTIME}`  | Name of the database  |
+| `--url` `-u`  | *string*  | `mongodb://localhost:27017`  | Mongo instance URL  |
```

Comment:
I'd rather remove these arguments from this main documentation page and instead only mention `yarn benchmark --help` (hoping it works).

Reply:
Now outputs as below. Can exit with a code other than `0` if you like.

![Screenshot 2021-07-06 140908](https://user-images.githubusercontent.com/10538031/124605425-68687b00-de5b-11eb-8094-36d5ae43fab5.png)